### DISCUSSION TITLE
Vercel Turbo version

### DISCUSSION THREAD

**Discussion by spacey-sooty** [vercel/community#7169] [Help] [answered]
### Summary

My GitHub actions CI build was failing due to turborepo 2.0 renaming pipeline to tasks. In order to fix this I renamed pipeline to tasks, but now my vercel build is failing with `unknown key tasks` 

### Example

https://github.com/CurtinFRC/Team-Website/pull/186 is where I encountered the issue

### Steps to Reproduce

1. create project with turborepo
2. attempt to deploy on vercel
3. get error
```
Error: turbo_json_parse_error
  x Found an unknown key `tasks`.
   ,-[2:1]
```

**Comment by versecafe** [comment, marked as answer]
@spacey-sooty You don't have a declared turbo dep in your root `package.json` so Vercel is likely using some default being v1 to not break other peoples systems yet. If you just add `"turbo": "^2.0.3",` to the `devDependencies` it should fix the error.

**Reply by spacey-sooty** [reply]
Thanks that did in fact fix it!

**Comment by swarnava** [comment]
Hi,

Please follow the upgrade instruction here: https://turbo.build/repo/docs/crafting-your-repository/upgrading

[a comment by an automated account is not shown]
